### DISCUSSION TITLE
Redis not working in Pimcore 10

### DISCUSSION THREAD

**Discussion by ravi-vegda** [pimcore/pimcore#9722] [Q&A / Support]
I have configure Redis as below in config\config.yaml
framework:
    cache:
        pools:
            pimcore.cache.pool:
                public: true
                tags: true
                default_lifetime: 31536000  # 1 year
                adapter: pimcore.cache.adapter.redis_tag_aware
                provider: 'redis://localhost:6379'
                
when I clear cache, it will show below error:

Cannot find the "redis" extension nor the "predis/predis" package: "redis://localhost:6379".

![image](https://user-images.githubusercontent.com/86904617/125239538-693f5800-e306-11eb-851a-0ceb10bac5f9.png)

**Comment by dvesh3** [comment]
@ravi-vegda Pimcore version?

**Reply by ravi-vegda** [reply]
Pimcore v10.0.7

**Reply by brusch** [reply]
You need to install redis extension for your PHP environment. 
e.g. `apt install php-redis`.
